docs/content/how-to/distribute-hosted-cluster-workloads.md

```diff
@@ -28,6 +28,41 @@ In addition:
       role.kubernetes.io/infra: "" 
 ```
 
+## Scheduling Topology Options 
+
+Cluster Service Providers may choose how hosted control planes are isolated or co-located. The three different options are:
+
+ - Shared Everything
+ - Shared Nothing 
+ - Dedicated Request Serving
+
+**NOTE**: Each hosted control plane can run Single Replica or Highly Available. If Highly Available, the control plane will be spread across failure domains via `topology.kubernetes.io/zone` as the topology key.
+
+### Shared Everything
+
+- All hosted control plane pods are scheduled to any node that can run hosted control plane workloads.
+
+![shared_everything_topology.png](..%2Fimages%2Fshared_everything_topology.png)
+
+### Shared Nothing
+
+- Nodes meant for a specific hosted cluster are tainted (and labeled) with a specific `hypershift.openshift.io/cluster` value.
```

Comment:
whos/whats responsible of setting the taint/label `hypershift.openshift.io/cluster` on the nodes?

Reply:
The label and taint are set during the dedicated request serving node reconciliation by the scheduler controller in the HO.

Reply:
so one would need to set the `hypershift.openshift.io/topology: dedicated-request-serving-components` annotation for 'Shared Nothing` option? if yes, how does it differ from the `Dedicated Request Serving` option.

Reply:
Oh sorry, I misread though you were asking about dedicated request serving topology. AFAIK you need to set the label and taint yourself.

Reply:
oh ok, can we then change the wording to make more clear, that the user/consumer should be setting the label/taint